webserver/grpc/src/main/java/io/helidon/webserver/grpc/GrpcProtocolHandler.java

```diff
@@ -167,6 +168,7 @@ public Http2StreamState streamState() {
     public void rstStream(Http2RstStream rstStream) {
         callCancelled = (rstStream.errorCode() == Http2ErrorCode.CANCEL);
         listener.onCancel();
+        currentStreamState = nextStreamState(Http2StreamState.HALF_CLOSED_REMOTE);
     }
```

Comment:
I don't think we have rstStream actually connected to anything yet, but when we do, it would be invoked from connection thread right?